python/web/list.py

```diff
@@ -0,0 +1,110 @@
+
+import cgi
+import glob
+from html.parser import HTMLParser
+
+from .index import capture_output
+
+
+head = '<html>' \
```

Comment:
Triple-quoted strings will work way better here, methinks.  That's more readable and maintainable.